### PR TITLE
Added app tittle and set parking lots button to activity_main.xml

### PR DESCRIPTION
**Pull request by Faka-GLB**
activity_main.xml was edited to add the app title TextView and a set parkings Button, in addition base_text_style was added to styles.xml resource and main_activity_button_inset to dimens.xml

image

image

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -4,15 +4,29 @@
...
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
```
Comment (mauroLonghi) [inline app/src/main/res/layout/activity_main.xml]: mmm if you have only two component, why do you put anohter layout? Choose put all in the contraint or use a simple linear layout

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -4,15 +4,29 @@
...
 
+        <TextView
+            android:id="@+id/textView_main_tittle"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@string/main_activity_app_tittle"
+            style="@style/base_text_style"
+            />
```
Comment (mauroLonghi) [inline app/src/main/res/layout/activity_main.xml]: please put in the previous sentence the close ->  style="@style/base_text_style" />

File: app/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="base_text_style" parent="TextAppearance.AppCompat.Display1">
```
Comment (mauroLonghi) [inline app/src/main/res/values/styles.xml]: the code style from style said that Themes goes in Cammel case

File: app/src/main/res/values/styles.xml [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+        <item name="android:textColor">@color/purple_700</item>
+        <item name="android:textAlignment">center</item>
+        <item name="android:textSize">20sp</item>
```
Comment (BQuinteros) [inline app/src/main/res/values/styles.xml]: pls remove this hardcode, you need to put it in the dimens file

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -1,3 +1,5 @@
 <resources>
     <string name="app_name">ParkingSystemJava</string>
+    <string name="main_activity_app_tittle">EstacionAR</string>
```
Comment (mauroLonghi) [inline app/src/main/res/values/strings.xml]: the app title and the app lenaguage its different with the app_name, its rally minor issue, but use the code style principles

**Comment by BQuinteros** [comment]
pls the next commits try to make it in english

File: app/src/main/AndroidManifest.xml [hunk 6]
```diff
@@ -9,7 +9,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.ParkingSystemJava">
-        <activity android:name=".MainActivity">
+        <activity android:name=".mvp.MainActivity">
```
Comment (SartiMau) [inline app/src/main/AndroidManifest.xml]: is this fine?

File: app/src/main/res/layout/activity_main.xml [hunk 7]
```diff
@@ -4,15 +4,29 @@
...
+
+        <Button
+            android:id="@+id/button_main_set_parkings"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:insetTop="@dimen/main_activity_button_inset"
+            android:insetBottom="@dimen/main_activity_button_inset"
+            android:text="@string/main_activity_button_set_parking" />
+    </LinearLayout>
 </androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: add an empty line at the end of the file to avoid this error. Check this in all the files

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (Faka-GLB) [reply app/src/main/res/layout/activity_main.xml]: Noted, the design interface didn't let me use a linear layout as a base layout, so I put a the linear layout inside the constraint layout, I'll fix this just by editing the .xml

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (Faka-GLB) [reply app/src/main/res/layout/activity_main.xml]: Will do.

File: app/src/main/res/values/styles.xml (hunk 3 above)
Comment (Faka-GLB) [reply app/src/main/res/values/styles.xml]: Noted, I'll change it to camel case.

File: app/src/main/AndroidManifest.xml (hunk 6 above)
Comment (Faka-GLB) [reply app/src/main/AndroidManifest.xml]: I made several reordering of packages, trying to imitate the mvpCounter project.

File: app/src/main/res/values/styles.xml (hunk 4 above)
Comment (Faka-GLB) [reply app/src/main/res/values/styles.xml]: can you elaborate?

File: app/src/main/res/layout/activity_main.xml (hunk 7 above)
Comment (Faka-GLB) [reply app/src/main/res/layout/activity_main.xml]: what's the error here? I don't understand

Comment (Faka-GLB) [reply app/src/main/res/layout/activity_main.xml]: Braian explained me, I'll add an empty line to the files.

File: app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -1,3 +1,5 @@
 <resources>
-    <string name="app_name">ParkingSystemJava</string>
-</resources>
\ No newline at end of file
+    <string name="app_name">estacionAR</string>
+    <string name="main_activity_textview_tittle">EstacionAR</string>
+    <string name="main_activity_button_setParking">Establecer estacionamientos</string>
```
Comment (BQuinteros) [inline app/src/main/res/values/strings.xml]: this name is not correct... this should be main_activity_button_set_parking. You are mixing snake case and camel case

File: app/src/main/AndroidManifest.xml [hunk 9]
```diff
@@ -9,7 +9,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.ParkingSystemJava">
-        <activity android:name=".MainActivity">
+        <activity android:name="com.onboarding.estacionAR.activities.MainActivity">
```
Comment (SartiMau) [inline app/src/main/AndroidManifest.xml]: here you can add only `.activities.MainActivity`

File: app/src/main/java/com/onboarding/estacionAR/activities/MainActivity.java [hunk 10]
```diff
@@ -1,14 +1,16 @@
-package com.onboarding.parkingsystemjava;
+package com.onboarding.estacionAR.activities;
```
Comment (SartiMau) [inline app/src/main/java/com/onboarding/estacionAR/activities/MainActivity.java]: the packages will be always in the singular. The correct is **activity** not **activities**

File: app/src/main/res/values/dimens.xml [hunk 11]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="main_activity_buttonInset">10dp</dimen>
+    <dimen name="textSize_strings_20sp">20sp</dimen>
```
Comment (SartiMau) [inline app/src/main/res/values/dimens.xml]: - Use always only one name style, snake_caso or camelCase, the resources will be con scnake_case.
- Use more representative names
- Do not put the specific dime in the title

**Review by SartiMau — CHANGES_REQUESTED** [review]
Why did you change the base package project's name?

File: app/src/main/res/values/strings.xml [hunk 12]
```diff
@@ -1,3 +1,5 @@
 <resources>
-    <string name="app_name">ParkingSystemJava</string>
-</resources>
\ No newline at end of file
+    <string name="app_name">estacionAR</string>
+    <string name="main_activity_textview_tittle">EstacionAR</string>
```
Comment (BQuinteros) [inline app/src/main/res/values/strings.xml]: same here this should be "main_activity_text_view_title"

File: app/src/main/res/layout/activity_main.xml [hunk 13]
```diff
@@ -1,18 +1,24 @@
...
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+<LinearLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical" >
+
+        <TextView
+            android:id="@+id/textView_main_tittle"
```
Comment (nicolbidone) [inline app/src/main/res/layout/activity_main.xml]: please use the naming convention of the course
https://docs.google.com/document/d/14nHjAQDKcPJS-Ns4UaHK6sHtB1ccbbhYq6nD_fIdmGU/edit#heading=h.hdrlward0n0v
https://jeroenmols.com/blog/2016/03/07/resourcenaming/
also the name should be descriptive

File: app/src/main/res/layout/activity_main.xml [hunk 14]
```diff
@@ -1,18 +1,24 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@string/main_activity_textview_tittle"
+            style="@style/baseTextStyle" />
+
+        <Button
+            android:id="@+id/button_main_set_parkings"
```
Comment (nicolbidone) [inline app/src/main/res/layout/activity_main.xml]: please use the naming convention of the course
 also the name should be descriptive

File: app/src/main/res/values/dimens.xml [hunk 15]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="main_activity_buttonInset">10dp</dimen>
```
Comment (nicolbidone) [inline app/src/main/res/values/dimens.xml]: please use the naming convention of the course
 also the name should be descriptive

File: app/src/main/res/values/strings.xml (hunk 8 above)
Comment (Faka-GLB) [reply app/src/main/res/values/strings.xml]: Done.

File: app/src/main/AndroidManifest.xml (hunk 9 above)
Comment (Faka-GLB) [reply app/src/main/AndroidManifest.xml]: Done.

File: app/src/main/java/com/onboarding/estacionAR/activities/MainActivity.java (hunk 10 above)
Comment (Faka-GLB) [reply app/src/main/java/com/onboarding/estacionAR/activities/MainActivity.java]: Acknowledged and changed.

File: app/src/main/res/values/dimens.xml (hunk 11 above)
Comment (Faka-GLB) [reply app/src/main/res/values/dimens.xml]: Done

File: app/src/main/res/values/strings.xml (hunk 12 above)
Comment (Faka-GLB) [reply app/src/main/res/values/strings.xml]: Done

File: app/src/main/res/layout/activity_main.xml (hunk 13 above)
Comment (Faka-GLB) [reply app/src/main/res/layout/activity_main.xml]: done

File: app/src/main/res/layout/activity_main.xml (hunk 14 above)
Comment (Faka-GLB) [reply app/src/main/res/layout/activity_main.xml]: done

File: app/src/main/res/values/dimens.xml (hunk 15 above)
Comment (Faka-GLB) [reply app/src/main/res/values/dimens.xml]: done

File: app/src/main/res/layout/activity_main.xml [hunk 16]
```diff
@@ -1,18 +1,24 @@
...
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+<LinearLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical" >
+
+        <TextView
+            android:id="@+id/text_main_tittle"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: pls fix the type.. it is title no tittle ;)

Comment (SartiMau) [reply app/src/main/res/layout/activity_main.xml]: and remember to format the code properly.. pls use ctrl + alt + l

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: build.gradle [hunk 17]
```diff
@@ -16,10 +16,9 @@ allprojects {
     repositories {
         google()
         mavenCentral()
-        jcenter() // Warning: this repository is going to shut down soon
```
Comment (SartiMau) [inline build.gradle:19]: is this change fine?

File: app/src/main/res/layout/activity_main.xml [hunk 18]
```diff
@@ -1,18 +1,24 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@string/main_activity_text_view_tittle"
```
Comment (nicolbidone) [inline app/src/main/res/layout/activity_main.xml]: mispelling 
main_activity_text_view_tittle
should be
main_activity_text_view_title

File: build.gradle (hunk 17 above)
Comment (Faka-GLB) [reply build.gradle:19]: yes, we've cheeked with lucho, and jcenter() will be deprecated, mavenCentral() now take care of that function, the IDE recommended to delete that line, works fine.

File: app/src/main/res/layout/activity_main.xml (hunk 16 above)
Comment (Faka-GLB) [reply app/src/main/res/layout/activity_main.xml]: done

File: app/src/main/res/layout/activity_main.xml (hunk 18 above)
Comment (Faka-GLB) [reply app/src/main/res/layout/activity_main.xml]: done

**Review by lucianoeli — COMMENTED** [review]
Looks good. aprove after comments.

File: app/src/main/res/layout/activity_main.xml [hunk 19]
```diff
@@ -1,18 +1,23 @@
...
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".MainActivity">
+    android:orientation="vertical">
 
     <TextView
-        android:layout_width="wrap_content"
+        android:id="@+id/text_main_tittle"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: The typo still here

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_main.xml (hunk 19 above)
Comment (Faka-GLB) [reply app/src/main/res/layout/activity_main.xml]: fixed

**Review by BQuinteros — APPROVED** [review]

**Review by SartiMau — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]

**Review by nicolbidone — APPROVED** [review]

**Review by lucianoeli — APPROVED** [review]

**Review by lucianoeli — APPROVED** [review]

**Pull request merged**